app/dao/fact_notification_status_dao.py

```diff
@@ -130,20 +130,27 @@ def update_fact_notification_status(data, process_day, service_ids=None):
 
 
 def fetch_notification_status_for_service_by_month(start_date, end_date, service_id):
+    filters = [
+        FactNotificationStatus.service_id == service_id,
+        FactNotificationStatus.bst_date >= start_date.strftime("%Y-%m-%d"),
+        # This works only for timezones to the west of GMT
+        FactNotificationStatus.bst_date < end_date.strftime("%Y-%m-%d"),
+        FactNotificationStatus.bst_date != datetime.utcnow().date().strftime("%Y-%m-%d"),  # Exclude current day
```

Comment:
Did you mean to have this here and in the FF check below?

Reply:
I did not, good catch - fixed!